pallet-logion-loc/src/lib.rs

```diff
@@ -207,6 +207,15 @@ pub type BalanceOf<T> = <<T as Config>::Currency as Currency<<T as frame_system:
 
 pub type NegativeImbalanceOf<T> = <<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId, >>::NegativeImbalance;
 
+#[derive(Encode, Decode, Default, Clone, PartialEq, Eq, Debug, TypeInfo)]
+pub struct Sponsorship<AccountId, EthereumAddress> {
+    sponsor: AccountId,
+    sponsored_account: SupportedAccountId<AccountId, EthereumAddress>,
+    legal_officer: AccountId,
+    used: bool,
```

Comment:
The flag will be turned to `true`upon first use (i.e. first doc publication), which imply a new sponsorship need to be created for each doc. It maybe difficult to know upfront how many docs will be required for a given ID LOC.

Reply:
Well, I was not seeing it that way: the `used` flag turns true upon LOC creation, not item publication.

Reply:
OK I misunderstood the usage of the flag, thanks for the clarification.